src/z3c/autoinclude/api.py

```diff
@@ -26,3 +27,15 @@ def disable_plugins():
 
 def enable_plugins():
     del os.environ[PLUGIN_KEY]
+
+
+def debug_enabled():
+    return DEBUG_KEY in os.environ
+
+
+def disable_debug():
+    os.environ[DEBUG_KEY] = 'True'
+
+
+def enable_debug():
+    del os.environ[DEBUG_KEY]
```

Comment:
These seem reversed, i.e. enalbe_debug() disables it, disable_debug() enables it?

Also, `del os.environ[DEBUG_KEY]` will raise an exception if DEBUG_KEY is not present in os.environ.  Consider try/except or `os.environ.pop(DEBUG_KEY, None)`.

Reply:
Ah, right, the debug case is the opposite of the others in this file. Plugins and dependencies are by default enabled, and debug by default disabled.

The KeyError was there in the existing functions too. I have fixed them and added simple tests for all of them now, and call everything twice, to ensure that this bug is tested.